packages/next/src/views/CreateFirstUser/index.client.tsx

```diff
@@ -13,9 +14,29 @@ export const CreateFirstUserFields: React.FC<{
 
   const fieldMap = getFieldMap({ collectionSlug: userSlug })
 
+  const firstUserFieldMapWithPasswordFields = createFirstUserFieldMap.map((field) => {
+    if (field.name === 'password') {
+      const type: keyof FieldTypes = 'password'
```

Comment:
You shouldn't have to type this here as long as the return value of the map is properly typed. You could put the  entire `type` property directly in the return statement. I'm also thinking this entire map here is unnecessary. We're already looping over fields to create `createFirstUserFieldMap` in the first place. It might be better to transform these fields to password in the first place.

Reply:
I moved it to the server, I think you're right.

The issue is if I don't put the type property it's not correctly inferred as a typeof FieldTypes and errors thinking that `type` is just a string.

Can't transform it before because `password` is not a valid `Field`